### ISSUE TITLE
JS Error when pasting a post (shortcode related)

### ISSUE THREAD

**Issue by youknowriad** [WordPress/gutenberg#13527]
**Describe the bug**

**To Reproduce**
 - Try pasting the following post into the visual editor of Gutenberg 
https://gist.github.com/florianbrinkmann/4e9e4d23eaefb8b23484badddd848196
 - Notice a JS error on the console

**Screenshots**

<img width="531" alt="capture d ecran 2019-01-26 a 5 24 59 pm" src="https://user-images.githubusercontent.com/272444/51790073-063ac480-2191-11e9-8c04-572c755134aa.png">

cc @iseulde @jorgefilipecosta

**Comment by wpaustria** [comment]
This issue came up during that discussion:
https://github.com/WordPress/gutenberg/issues/11782
Here are the videos which belong to this issue:

- https://www.youtube.com/watch?v=lttL0TxPFbU&feature=youtu.be
- https://www.youtube.com/watch?v=HwFnG77QOvk&feature=youtu.be
- https://www.youtube.com/watch?v=gTMHEAs68Mc&feature=youtu.be
- https://www.youtube.com/watch?v=W-X6hPB4OfY&feature=youtu.be

**Comment by jorgefilipecosta** [comment]
I'm having problems reproducing the JS error, (although the pasting of this content is very slow). @wpaustria what software are you copying the content from? Does the JS error also happen if the content is pasted using cmd+shift+v (or equivalent in your keyboard)? Thank you for all the help debugging this issue.

**Comment by youknowriad** [comment]
@jorgefilipecosta I reproduced this error by copying the whole content, clicking the appender, and pasting.

**Comment by wpaustria** [comment]
@jorgefilipecosta As shown in the videos, I copied the content with CMD+C/CMD+V and with the context menu. Both ways don't work. 
Software: Firefox 64 & latest Chrome & latest Chromium on Ubuntu 18.10.
I copied the content from a website, have a look:

https://gist.github.com/florianbrinkmann/4e9e4d23eaefb8b23484badddd848196

This is called the infamous "The Florian Brinkmann article". 
Which shows copying is very slow. Before the latest changes copying this content to Gutenberg was impossible!
